README.md

```diff
@@ -88,26 +88,22 @@ The steps required are outlined below.
 - Run the build script appropriate for your environment.
      - **Windows:** [build.cmd](https://github.com/dotnet/templating/blob/master/build.cmd)
      - **Mac/Linux**: [build.sh](https://github.com/dotnet/templating/blob/master/build.sh) 
-- When running the build script, the existing built-in command `dotnet new` will be preserved. A new command `dotnet new3` will be enabled which allows you to create
-files with the latest Template Engine.
-- That's it! Now you can run `dotnet new3`.
+- When running the build script, the existing built-in command `dotnet new` will be preserved. To run `dotnet new3` run `dotnet <your repo location>\artifacts\bin\dotnet-new3\Debug\netcoreapp2.1\dotnet-new3.dll` (root path to `dotnet-new3.dll` is skipped in all commands below).
 
-For example, here is the result of running `dotnet new3 --help` on a Mac (_truncated to save space here_).
+For example, here is the result of running `dotnet .\dotnet-new3.dll --help` (_truncated to save space here_).
 
 ```bash
-$ dotnet new3 --help
-Template Instantiation Commands for .NET Core CLI.
+$ dotnet .\dotnet-new3.dll --help
+Usage: new3 [options]
 
-Usage: dotnet new3 [arguments] [options]
-
-Arguments:
-  template  The template to instantiate.
-
-<truncated>
+Options:
+  -h, --help          Displays help for this command.
+  -l, --list          Lists templates containing the specified template name. If no name is specified, lists all templates.
+  -n, --name          The name for the output being created. If no name is specified, the name of the output directory is used.
+...
 ```
-### Template NuGet Packages
-- The build that produces the **template NuGet packages** currently has a dependency on **nuget.exe**.
-  - Because of this, those that wish to `install` using the **template NuGet packages** will need to be on Windows in order to produce the appropriate assets. 
+After first installation there is no templates installed. See [Installing templates](#Installing-templates) on how to install the templates and [Available templates](#Available-templates).
```

Comment:
In first sentence, should be `there *are* no templates installed`

Reply:
Having "Available templates" at the end, second sentence sounds unfinished. Could you read once more and see if it sounds right?